.travis.yml

```diff
@@ -19,10 +19,10 @@ services: docker
 
 env:
   matrix:
-  - INSTANCE=server-centos-6
-  - INSTANCE=server-centos-7
-  - INSTANCE=server-ubuntu-1404
-  - INSTANCE=server-ubuntu-1604
+  - INSTANCE=default-centos-6
+  - INSTANCE=default-centos-7
+  - INSTANCE=default-ubuntu-1404
+  - INSTANCE=default-ubuntu-1604
```

Comment:
if Fedora / openSUSE work we should get them in here. It's not a blocker on 3.0 though